symplyphysics/laws/hydro/volume_of_the_submerged_body_part_from_body_density_and_liquid_density.py

```diff
@@ -34,6 +35,7 @@ def print_law() -> str:
 @validate_output(submerged_volume)
 def calculate_submerged_volume(body_volume_: Quantity, body_density_, liquid_density_: Quantity) -> Quantity:
     result_expr = solve(law, submerged_volume, dict=True)[0][submerged_volume]
+    assert simplify(body_density_ - liquid_density_) < 0

```

Comment:
Assert is not the best way to throw an exception. Check other parts of the code to find an example of throwing an exception due to invalid parameters.